docs/rc.xml.all

```diff
@@ -99,6 +99,9 @@
   <snapping>
     <!-- Set range to 0 to disable window snapping completely -->
     <range>1</range>
+		<!-- edgeType is optional -->
+		<previewDelay edgeType="interior">500</previewDelay>
+		<previewDelay edgeType="exterior">500</previewDelay>
```

Comment:
`rc.xml.all` uses spaces for some history reason

Reply:
Done.
The problem is VSCode by default uses `.vscode/settings.json` and ignores `.editorconfig`.
Maybe I'll make a PR to add `.vscode/settings.json` or mention it in `CONTRIBUTING.md`.

Reply:
Still a tab in front on the latest fixup

Reply:
Done. Sorry for my oversight.